source/fuzz/fuzzer_pass_add_dead_continues.cpp

```diff
@@ -32,15 +32,41 @@ void FuzzerPassAddDeadContinues::Apply() {
   // Consider every block in every function.
   for (auto& function : *GetIRContext()->module()) {
     for (auto& block : function) {
+      // Get the label id of the continue target
+      // of the innermost loop.
```

Comment:
Nit: can you make this a single-line comment?

Reply:
Fixed in a04b185.